sklearn/ensemble/iforest.py

```diff
@@ -110,12 +110,22 @@ class IsolationForest(BaseBagging, OutlierMixin):
     offset_ : float
         Offset used to define the decision function from the raw scores.
         We have the relation: ``decision_function = score_samples - offset_``.
+        Assuming behaviour == 'new', offset_ is defined as follows.
         When the contamination parameter is set to "auto", the offset is equal
         to -0.5 as the scores of inliers are close to 0 and the scores of
         outliers are close to -1. When a contamination parameter different
         than "auto" is provided, the offset is defined in such a way we obtain
         the expected number of outliers (samples with decision function < 0)
         in training.
+        Assuming the behaviour parameter is set to 'old', we always have
+        offset_ = -0.5.
+
+    behaviour: str, optional (default='old')
+        Accepted values are 'old' or 'new'. Behaviour of the decision_function.
+        Default "behaviour" parameter will change to "new" in version 0.22.
+        Passing behaviour="new" makes the decision_function change to match
+        other anomaly detection algorithm API, as explained in the offset_
+        parameter documentation.
```

Comment:
I think that you should be slightly more explicit as to what switching this parameter entails.